content/learn/migration-guides/0.12-to-0.13.md

```diff
@@ -1231,7 +1232,7 @@ The increased `Exposure::ev100` means that all existing 3D lighting will need to
   - The trait now requires `Clone`.
   - The `ExtractedAsset` associated type has been removed (the type itself is now extracted).
   - The signature of `prepare_asset()` is slightly different
-  - A new `asset_usage()` method is now required (return `RenderAssetUsage::RENDER_WORLD` to match the previous behavior).
+  - A new `asset_usage()` method is now required (return `RenderAssetUsages::default()` to match the previous behavior).
```

Comment:
The original migration guide stated the `asset_usage` should return `RenderAssetPersistencePolicy::Unload` to match the previous `AssetRender` behavior, which maps to `RenderAssetUsage::RENDER_WORLD` and then unloading the asset from CPU.

By changing it to `RenderAssetUssages::default()` the asset will be kept both in CPU and GPU. This was a mistake on original PR or something has changed in another PR?

Reply:
Seems like a mistake in the original guide to me.